tests/test_frame.py

```diff
@@ -1012,6 +1016,12 @@ def test_frame_getitem_isin() -> None:
     assert_type(df[df.index.isin([1, 3, 5])], "pd.DataFrame")
 
 
+def test_read_excel() -> None:
```

Comment:
can you add a `pytest.skip()` here, as this will fail with `pytest`, since `foo` doesn't exist.

Also add a comment referring to the Github issue, e.g. `# GH 33`